ui-components/src/page_components/listing/AdditionalFees.tsx

```diff
@@ -5,13 +5,13 @@ export interface AdditionalFeesProps {
   depositMin?: string
```

Comment:
So this is the piece where it also needs to happen in core. We're trying to remove all changes in ui-components that are not in core so that we can centralize on one version. I think it's completely fine to make these ui-component changes in Detroit first, and have a separate PR in core after this is merged to bring these changes back in.